assets/src/edit-story/components/inspector/prepublish/prepublishChecklistProvider.js

```diff
@@ -75,7 +75,11 @@ function PrepublishChecklistProvider({ children }) {
   // 1. Story is no longer empty (ON_DIRTY_STORY)
   // 2. Publish button is hit on a draft (ON_PUBLISH_CLICKED)
   // 3. Story has more than 4 pages
+  // 4. Story has more than 2 pages
   useEffect(() => {
+    if (story.pages.length > 1) {
```

Comment:
should we just listen to the `onSecondPageAdded` event here since we already created it? I thought this effect was just a stopgap until we had all the story triggers/events done

Reply:
yesssssssssssss you're right